api/backend.go

```diff
@@ -485,6 +486,52 @@ func (b *StatusBackend) SendDataNotification(dataPayloadJSON string, tokens ...s
 	return err
 }
 
+func (b *StatusBackend) InjectChatAccount(chatKeyHex, encryptionKeyHex string) error {
+	b.mu.Lock()
+	defer b.mu.Unlock()
+
+	b.accountManager.Logout()
+
+	chatKey, err := ethcrypto.HexToECDSA(chatKeyHex)
+	if err != nil {
+		return err
+	}
+
+	err = b.accountManager.InjectChatAccount(chatKey)
+	if err != nil {
+		return err
+	}
+
+	chatAccount, err := b.accountManager.SelectedChatAccount()
+	if err != nil {
+		return err
+	}
+
+	whisperService, err := b.statusNode.WhisperService()
```

Comment:
Any interaction with Whisper is done here because we use the same key pair for chat and Whisper still, right? If so, let's add a comment explaining that.

Reply:
If it's the same code, let's extract it to a separate method 👍